content/tutorials/manage-the-rippled-server/cluster-rippled-servers.md

```diff
@@ -1,22 +1,87 @@
 # Cluster rippled Servers
 
-If you are running multiple `rippled` servers in a single datacenter, you can configure those servers into a cluster to maximize efficiency. Running your `rippled` servers in a cluster provides the following benefits:
+If you run multiple `rippled` servers in the same data center, you can configure them in a [cluster](clustering.html) to maximize efficiency. To configure clustering:
 
-* Clustered `rippled` servers share the work of cryptography. If one server has verified the authenticity of a message, the other servers in the cluster trust it and do not re-verify.
-* Clustered servers share information about peers and API clients that are misbehaving or abusing the network. This makes it harder to attack all servers of the cluster at once.
-* Clustered servers always propagate transactions throughout the cluster, even if the transaction does not meet the current load-based transaction fee on some of them.
+1. For each of your servers, note the IP address of the server.
 
-To enable clustering, change the following sections of your [config file](https://github.com/ripple/rippled/blob/master/cfg/rippled-example.cfg) for each server:
+2. For each of your servers, generate a unique seed using the [validation_create method][].
 
-* List the IP address and port of each other server under the `[ips_fixed]` section. The port should be the one from the other servers' `protocol = peer` setting in their `rippled.cfg`. Example:
+    For example, using the commandline interface:
 
-        [ips_fixed]
-        192.168.0.1 51235
-        192.168.0.2 51235
+        $ rippled validation_create
+
+        Loading: "/etc/rippled.cfg"
+        Connecting to 127.0.0.1:5005
+        {
+           "result" : {
+              "status" : "success",
+              "validation_key" : "FAWN JAVA JADE HEAL VARY HER REEL SHAW GAIL ARCH BEN IRMA",
+              "validation_public_key" : "n9Mxf6qD4J55XeLSCEpqaePW4GjoCR5U1ZeGZGJUCNe3bQa4yQbG",
+              "validation_seed" : "ssZkdwURFMBXenJPbrpE14b6noJSu"
+           }
+        }
+
+    Save the `validation_seed` and `validation_public_key` parameters from each response somewhere secure.
+
+3. On each server, edit the [config file](https://github.com/ripple/rippled/blob/master/cfg/rippled-example.cfg), modifying the following sections:
+
+    1. List the IP address and port of each other server under the `[ips_fixed]` section. The port should be the one from the other servers' `protocol = peer` setting in their `rippled.cfg` (usually 51235). Each member of the cluster should list all _other_ members of the cluster here. For example:
```

Comment:
nit: Since the introductory step says "modifying the following sections:" maybe start the nested step with "Under the `[ips_fixed]` section, list the IP address..." and so on so it tracks a little more, e.g.

2. In the `[node_seed]` section, set the server's node seed...
3. In the `[cluster_nodes]` section...